downloaders/apkmirror-downloader/build.gradle.kts

```diff
@@ -0,0 +1,16 @@
+plugins {
+    alias(libs.plugins.kotlin.parcelize)
+}
+
+android {
+    val packageName = "app.revanced.manager.plugin.downloader.apkmirror"
+    namespace = packageName
+    defaultConfig {
+        applicationId = packageName
+    }
+
+    dependencies {
+        implementation(project(":arsclib"))
```

Comment:
Is this dependency necessary? `shared` already depends on it. Same goes for the play store downloader.